core-backend/sandbox/src/runtime.rs

```diff
@@ -21,18 +42,22 @@ pub(crate) fn as_i64(v: Value) -> Option<i64> {
     }
 }
 
-pub(crate) struct Runtime<E: Ext> {
+pub(crate) struct Runtime<E: Ext + IntoExtInfo<E::Error>> {
```

Comment:
Sorry, but see this extra bound second time here
From where it appears, why do we need this?

Reply:
Because now pre_process_memory_accesses is in IntoExtInfo, because @ark0f  ask me to do this.